app.js

```diff
@@ -0,0 +1,11 @@
+const express = require("express");
+const { getAllTopics } = require("./controllers/news.controller");
+const { error_handler } = require("./error-handlers/error-handler");
+const app = express();
+
+app.use(express.json());
```

Comment:
We aren't posting data yet, this doesn't need to be here. Keep to only the code you need for each ticket.